.drone.yml

```diff
@@ -39,8 +43,6 @@ steps:
         from_secret: CLOUDFRONT_KEY_PAIR_ID
       CLOUDFRONT_PRIVATE_KEY:
         from_secret: CLOUDFRONT_PRIVATE_KEY
-      CODECOV_TOKEN:
-        from_secret: CODECOV_TOKEN
```

Comment:
I believe that's a [Drone secret](https://docs.drone.io/secret/repository/)

Reply:
ah, and there's a comment right here https://github.com/code-dot-org/code-dot-org/blob/75038ab9374fa219f4b5cdf9712f74d8c948a475/.drone.yml#L41 that says we source the drone secrets from https://github.com/code-dot-org/code-dot-org/blob/da1f92c7e05ac48fa70e93b46f3c024cb0aa6906/docker/unit_tests.sh#L19-L41

I do note that script is appending to `locals.yml` rather than simply writing to it, so it's possible something else is writing that value somewhere. But if anything is, it's doing so from outside this codebase and doesn't need to be dealt with as part of this PR.